_pages/ValinaClearwater.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: ValinaClearwater
+pageUrl: ValinaClearwater.github.io
+timestamp: 2017-04-13
+---
```

Comment:
Hey @valinaclearwater ! You're so close! Just add a new line to the end of this file and your build should pass. 

After that change, you should be all set! 😄 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.